setup.cfg

```diff
@@ -20,22 +20,21 @@ license_files = LICENSE
 # TODO(xuanwn): include all files in src/python/grpcio/grpc
 [pytype]
 inputs =
-    src/python/grpcio/grpc/experimental
```

Comment:
I believe I've left this comment twice before on other reviews, but if we're removing coverage for existing code in this PR, then we need to have a very prominent TODO comment noting that we cannot merge the dev branch to master before the coverage is restored.